--- ./Makefile.in	2013-06-14 12:10:15.266629528 +0200
+++ ./Makefile.in	2013-06-14 12:10:52.434637900 +0200
@@ -389,7 +389,7 @@
 top_build_prefix = @top_build_prefix@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-SUBDIRS = pango modules pango-view examples docs tools tests build
+SUBDIRS = pango modules pango-view examples docs tools build
 EXTRA_DIST = \
 	autogen.sh		\
 	pango.pc.in		\

--- ./Makefile.am	2013-06-14 12:06:52.026642887 +0200
+++ ./Makefile.am	2013-06-14 12:07:19.762637103 +0200
@@ -1,6 +1,6 @@
 ## Process this file with automake to create Makefile.in.

-SUBDIRS= pango modules pango-view examples docs tools tests build
+SUBDIRS= pango modules pango-view examples docs tools build

 EXTRA_DIST = 			\
 	autogen.sh
